### PR TITLE
Changes to the system for featured profiles 

### LINKED ISSUE #1348: Featured profiles don't account for name changes
When a featured profile changes their name ingame, they no longer show up on the frontpage.

### PR DESCRIPTION
**Pull request by MartinNemi03**
- Changed featured profiles to use UUID instead of usernames
- Changed the system to export into a JSON file instead of a database

Fixes #1348

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -140,6 +140,8 @@ function updateCommitHash() {
 }
 const commitHash = updateCommitHash();
 
+const featuredProfiles = await fs.readJSONSync(path.resolve("./public/resources/js/featured-profiles.json"));
```
Comment (nstringham) [inline src/app.js]: why is there an `await` here?

**Comment by dukio** [comment]
But why is it needed to update them? nothing gets updated... it's just a static object no? 😕 🤔 

The whole `/scripts/update-featured-profiles.js)` seems useless to me, no?

**Comment by nstringham** [comment]
What about skins are those still going to auto update?

**Comment by dukio** [comment]
> What about skins are those still going to auto update?

Yeah they are from crafatar, so external

**Comment by nstringham** [comment]
> > What about skins are those still going to auto update?
> 
> Yeah they are from crafatar, so external

Good point

**Comment by MartinNemi03** [comment]
> But why is it needed to update them? nothing gets updated... it's just a static object no? 😕 🤔
> 
> The whole `/scripts/update-featured-profiles.js)` seems useless to me, no?

You need to update the usernames and emojis somehow. 
Also that's why it only runs on start, it only needs to get updated user info and that's it.

**Comment by dukio** [comment]
> > But why is it needed to update them? nothing gets updated... it's just a static object no? 😕 🤔
> > The whole `/scripts/update-featured-profiles.js)` seems useless to me, no?
> 
> You need to update the usernames and emojis somehow. Also that's why it only runs on start, it only needs to get updated user info and that's it.

oh right didn't think of that

**Comment by dukio** [comment]
#1358

File: src/app.js (hunk 1 above)
Comment (dukio) [reply src/app.js]: Agree it doesn't look like the await is needed: https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/readJson-sync.md

**Review by dukio — APPROVED** [review]
👍

**Pull request merged**